2020-11-16 13:35:01,406 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 1826372ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 19:22:20,458 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 2850678ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 19:37:26,775 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 886794ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 20:24:34,451 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 2307618ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 20:28:40,565 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 246111ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 20:39:18,118 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 637546ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 20:54:31,459 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 913335ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 22:02:04,032 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 292061ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-18 17:13:30,921 ERROR 4120 nodejs.ClusterClientNoResponseError: client no response in 1457366ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4120
hostname: fanhongkaideMacBook-Pro.local

2020-11-18 18:35:02,554 ERROR 4120 nodejs.ClusterClientNoResponseError: client no response in 1837398ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4120
hostname: fanhongkaideMacBook-Pro.local

2020-11-18 19:21:17,291 ERROR 4120 nodejs.ClusterClientNoResponseError: client no response in 1974541ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4120
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 00:39:38,821 ERROR 13535 [-/127.0.0.1/-/18ms POST /login] nodejs.TypeError: ctx.service.user.queryRoleInfoByRoleId is not a function
    at HomeController.login (/Volumes/projects/restaurant-admin-service/app/controller/home.js:53:51)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 13535
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 00:40:14,902 ERROR 13567 [-/127.0.0.1/-/19ms POST /login] nodejs.TypeError: ctx.service.user.queryRoleInfoByRoleId is not a function
    at HomeController.login (/Volumes/projects/restaurant-admin-service/app/controller/home.js:53:51)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 13567
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 00:40:31,068 ERROR 13579 [-/127.0.0.1/-/17ms POST /login] nodejs.TypeError: ctx.service.user.queryRoleInfoByRoleId is not a function
    at HomeController.login (/Volumes/projects/restaurant-admin-service/app/controller/home.js:53:51)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 13579
hostname: fanhongkaideMacBook-Pro.local

